webapps/api/backend/views/vdisks.py

```diff
@@ -118,6 +118,10 @@ def set_config_params(self, vdisk, new_config_params, version):
                 raise HttpNotAcceptableException(error_description='API version 1 requires a Storage Router IP',
                                                  error='invalid_version')
             new_config_params['dtl_target'] = [junction.domain_guid for junction in storage_router.domains]
+        if version >= 6:
```

Comment:
Why would we need this check? We just check if the values are in there and remove them, even with newer clients (where they are not in there anyway). And since the VDiskController is always the latest code, it should always be removed.